README-menus.md

```diff
@@ -0,0 +1,223 @@
+# Overview
+This README's purpose is to guide users through setting up Discord GUI menus with Clearingway.  
+All things discussed here are only relevant to `config.yaml`.
+# Menus
+There are currently 4 types of menus:
+- `menuMain`
+- `menuVerify`
+- `menuRemove`
+- `menuEncounter`
+
+If the menus module is enabled, there will be 3 menus that will be configured by default, all named by their namesake types: `menuMain`, `menuVerify`, `menuRemove`. 
+## Common
+All menus will be able to utilize these properties.  
+#### Required properties
+- `name` (string) - The internal name of the menu. Using the aforementioned default menu names will allow you to overwrite defaults for the title, description, etc. Otherwise, it will create a new menu.  
+- `type` (string) - The menu type with 4 possible values
+    - `menuMain`
+    - `menuVerify`
+    - `menuRemove`
+    - `menuEncounter`
+
+For creating new menus, the only types to be concerned with are `menuMain` and `menuEncounter`. `menuVerify` and `menuRemove` should only be used to override the default text/style properties.
+
+##### Optional properties
+- `title` (string) - The title of the embed
+- `description` (string) - The description of the embed
+- `imageUrl` (string) - A full-size image to be displayed at the bottom of the embed
+- `thumbnailUrl` (string) - A thumbnail embed to be displayed at the top-right of the embed
+- `fields` (array of object) - Fields to be displayed within an embed  
+  - `name` (string) - Title of field
+  - `value` (string) - Description of field
+  - `inline` (bool) - Whether field is inline or not
+
+## MenuMain
+Menus that are sendable to the channel which are viewable by every user
+#### Optional properties
+##### `buttons` (array of object)
+A list of buttons to be displayed under the embed. Each button object has the following required properties:
+- `label` (string) - Button label
+- `style` (int) - Button color value, follows `discordgo.ButtonStyle` enums
+  - `1` = Primary (Blurple)
+  - `2` = Secondary (Grey)
+  - `3` = Success (Green)
+  - `4` = Danger (Red)
+  - `5` = URL (Grey), prob shouldn't be used?
```

Comment:
let's make this into a statement sand say "we recommend not using this" or omit the bullet point entirely